### ISSUE TITLE
VS 2015 Update 3 - complete freeze / crash

### ISSUE THREAD

**Issue by Robert969696** [dotnet/roslyn#14177]
**Envinorment: Windows 10, Visual Studio 2015 Enteprise with Update 2**: 

Problem came out today when i updated from **Update 2 to Update 3**. Visual studio just crashes after a while when i open any project. I mean its just starts to freeze completly. I found resolution on this topic: https://github.com/dotnet/roslyn/issues/12317 . I've installed mentioned fix but didn't work. What i did next was to repair VS in control panel - didn't work. Then i uinstalled completly and then again installed again but still i got the same issue. I cannot even develop ! What can i do above that? Its just get freezes and then restatrt VS.

I checked also my windows 10' event viewer and found something like this:

> The application name causes confusion: devenv.exe, version: 14.0.25420.1, timestamp: 0x57685d85 The name of the module causing the error: mscorlib.ni.dll version: 4.6.1586.0, time stamp: 0x575a1299 Exception code: 0xc00000fd The offset error 0x0039f4e3 Process ID causing the error: 0x38e4 Hour run the application causing the error: 0x01d219f6057461d5 The application path causing the error: C: \ Program Files (x86) \ Microsoft Visual Studio 14.0 \ Common7 \ IDE \ devenv.exe The path of the module causing the error: C: \ WINDOWS \ assembly \ NativeImages_v4.0.30319_32 \ mscorlib \ 40571abae9422cd2ca6fafbbde1c3cdc \ mscorlib.ni.dll Report ID: 0956078b-1b3d-4b24-bac7-3f39cc4c53ea The full name of the package causing the error: The application ID terms of the package causing the error:

**Comment by dsaf** [comment]
I have same 2015.3 version and it was constantly crashing with any version of ReSharper installed. There seems to be big problems with the new ASP.NET Core projects - lack of TFS compatibility, general stability issues.

**Comment by vbcodec** [comment]
try run with /ResetSetting, but that clear all settings, or try /SafeMode. If this do not help then I suggest to uninstall, delete all VS directories in program files and in c:/uses/youraccount/AppData and install again (best final version with update3). That what helped me to 'recover' from similar mess.

**Comment by Robert969696** [comment]
Suppose i already tried every fix/workaround published over internet. Nothing works so far. At the end i decided to install Windows 10 from sratch and install VS again. That worked as a charm. Now problem dissapeared. I lost 1,5 day to recover...  Conclusion? Never update straightaway after new relaase came in, i think its much more safety than install when it cames out. In my case i will never install update before half year after released.

**Comment by jasonmalinowski** [comment]
@Robert969696: if you look in the event viewer, alongside the "Application Error" event you should also see a "Windows Error Reporting" event, which has additional information we'll need to track this down on our side. Can you provide that information as well?

(tagging @tmeschter who is our error reporting guru.)

**Comment by Robert969696** [comment]
Would really like to get it for you and everyone else but as i sait i already installed brain new windows 10 copy therefore logs from old system has been during format.

**Comment by jasonmalinowski** [comment]
OK, if you see it again let us know, otherwise there's not much else for us to go on from here.

**Comment by neilsimp1** [comment]
I'm having the same issues, I believe.

Windows 10, VS Community 2015 Update 3, and a .Net Core project.Anytime I save a file, VS freezes and needs me to kill it and restart.

Application Error:

`Faulting application name: devenv.exe, version: 14.0.25420.1, time stamp: 0x57685d85`
`Faulting module name: clr.dll, version: 4.6.1586.0, time stamp: 0x575a139f`
`Exception code: 0xc00000fd`
`Fault offset: 0x0003c9af`
`Faulting process id: 0x1f4c`
`Faulting application start time: 0x01d227d9f849bb7a`
`Faulting application path: C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\devenv.exe`
`Faulting module path: C:\Windows\Microsoft.NET\Framework\v4.0.30319\clr.dll`
`Report Id: cb873477-89cb-4c49-8c8d-d878014ab535`
`Faulting package full name: `
`Faulting package-relative application ID: `


Windows Error Reporting:

`Fault bucket 108273193046, type 1`
`Event Name: APPCRASH`
`Response: Not available`
`Cab Id: 0`

`Problem signature:`
`P1: devenv.exe`
`P2: 14.0.25420.1`
`P3: 57685d85`
`P4: clr.dll`
`P5: 4.6.1586.0`
`P6: 575a139f`
`P7: c00000fd`
`P8: 0003c9af`
`P9: `
`P10: `

`Attached files:
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER2E23.tmp.WERInternalMetadata.xml`

`These files may be available here:
C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_devenv.exe_b6555866f2a422a58783fe3967395565b84beecc_12aa736e_1b423eec`

`Analysis symbol: `
`Rechecking for solution: 0`
`Report Id: a675dec3-bb73-4f20-9fa8-9dd150291be6`
`Report Status: 0`
`Hashed bucket: 43c5dec0d581e34f36cd95500dcdd847`

**Comment by jasonmalinowski** [comment]
@neilsimp1: that's probably not the same issue, but thanks for the report. I've tracked down the bug on an internal system, trying to figure out it's status.

**Comment by semuserable** [comment]
TL;DR Have the same issue, only reproducible on VS2015 after updating it from 'Update 2' to 'Update 3'.

I have 2 PCs: one with VS2015 'Update 2' and the other VS2015 with 'Update 3'. After updating my first PC VS2015 to 'Update 3' and opening Roslyn.sln, VS crashed on "Preparing Solution..." phase. I have no problems whatsoever opening it on the other PC with VS2015 'Update 3' built-in. 

Some logs
```
Last Write Time:	10/18/2016 1:19:50 PM
Process Name:	devenv.exe : C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\devenv.exe
Process Architecture:	x86
Exception Code:	0xC00000FD
Exception Information:	The thread used up its stack.
Heap Information:	Present
Error Information:

System Information
------------------
OS Version:	6.1.7601
CLR Version(s):	4.6.1055.0
```

**Comment by jasonmalinowski** [comment]
@semuserable to confirm if it's the same issue, we need the same information from the Application event log that @neilsimp1 provided. I'd encourage you to file a new bug in that case; it's easier for us to handle one-issue-per-bug (that _might_ be duplicates) than having to split duplicates out of one issue.

**Comment by jasonmalinowski** [comment]
@neilsimp1: this particular crash is
